src/components/job-overview/index.js

```diff
@@ -169,7 +171,7 @@ JobOverview.propTypes = {
   appName: PropTypes.string.isRequired,
   job: PropTypes.exact(jobModel),
   jobName: PropTypes.string.isRequired,
-  repo: PropTypes.string,
+  repo: PropTypes.string.isRequired,
```

Comment:
should this property be required? - there is a test in CommitHash component if repo property exist

Reply:
Agreed; this shouldn't be required